medical/README.rst

```diff
@@ -6,83 +6,43 @@
 Odoo Medical
 ============
 
-What is Odoo Medical
---------------------
-
-**Odoo Medical** is a multi-user, highly scalable, centralized Electronic Medical
-Record (EMR) and Hospital Information System for odoo.
-
-**Odoo Medical** provides a free universal Health and Hospital Information System,
-so doctors and institutions all over the world, specially in developing
-countries will benefit from a centralized, high quality, secure and scalable
-system.
-
-Odoo Medical at a glance:
--------------------------
-
-- Strong focus in family medicine and Primary Health Care
-
-- Major interest in Socio-economics (housing conditions, substance abuse, education...)
-
-- Diseases and Medical procedures standards (like ICD-10 / ICD-10-PCS ...)
-
-- Patient Genetic and Hereditary risks : Over 4200 genes related to diseases (NCBI / Genecards)
-
-- Epidemiological and other statistical reports
-
-- 100% paperless patient examination and history taking
-
-- Patient Administration (creation, evaluations / consultations, history ... )
-
-- Doctor Administration
-
-- Lab Administration
-
-- Medicine / Drugs information (vademécum)
-
-- Medical stock and supply chain management
-
-- Hospital Financial Administration
-
-- Designed with industry standards in mind
-
-- Open Source : Licensed under AGPL
+This module extends Odoo with the base functionality of medical patients and centers.
 
```

Comment:
Should you add a section"

Installation
========
You will need https://github.com/OCA/partner-contact